src/VehicleSetup/JoystickConfigAdvanced.qml

```diff
@@ -88,6 +88,23 @@ Item {
             }
         }
         //-----------------------------------------------------------------
+        //-- Zero throttle while not flying
+        QGCLabel {
+            text:               qsTr("Zero throttle while aircraft not flying")
+            Layout.alignment:   Qt.AlignVCenter
+            Layout.minimumWidth: ScreenTools.defaultFontPixelWidth * 36
+        }
+        QGCCheckBox {
+            id:         zeroThrottleNotFlying
+            checked:    activeVehicle.joystickMode !== 0
+            Component.onCompleted: {
+                checked = _activeJoystick.zeroThrottleNotFlying
+            }
+            onClicked: {
```

Comment:
`OnCheckedChanged` could be more generic. I don't know joystick modes could change and change the check without clicking.

Reply:
Changed